docs/docs/platforms/OPENSHIFT.md

```diff
@@ -47,6 +53,31 @@ volumes:
       emptyDir: {}
 ```
 
+While it's possible to declare runtime volumes for empty directories, it is not possible for `${APPLICATION_HOME}/conf`. When starting up, Jira and Confluence generate a few configuration files which is a part of image entrypoint. Without anyuid SCC, an unprivileged user can't write to `${APPLICATION_HOME}/conf`. When starting Jira in OpenShift without anyuid SCC attached to jira service account, you will see the following log:
```

Comment:
`Without any uid SCC`?

Reply:
SCC is called `anyuid` https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth

Reply:
I have pushed a commit with `anyuid` font to make sure it's clear it's the scc name